annotator/src/main/java/org/cbioportal/annotator/internal/GenomeNexusImpl.java

```diff
@@ -512,7 +512,7 @@ private String resolveConsequence() {
     }
     
     private String resolveEntrezGeneId(boolean replace) {
-        if (!replace) {
+        if (!replace || canonicalTranscript == null || canonicalTranscript.getGeneSymbol() == null || canonicalTranscript.getGeneSymbol().trim().length() == 0) {
```

Comment:
we want to check whether the `canonicalTranscript .geneId()` is null or empty here, not gene symbol (that's the ensembl gene accession that the api uses)